src/components/radios/index.md.njk

```diff
@@ -84,19 +84,23 @@ If one or more of your radio options is different from the others, it can help u
 
 ### Conditionally revealing content
 
-You can conditionally reveal content when the user selects a particular radio, so they only see content when it’s relevant to them.
+You can conditionally reveal a related question when the user selects a particular radio option, so they only see content when it’s relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them together. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
```

Comment:
Is 'together' redundant after 'grouping'?

Reply:
Hmm.. redundant, yes. Extraneous? Not sure!  

I was using 'together' to reinforce the point, but we could do this instead:
```suggestion
This might make two related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
```

Reply:
That suggestion looks good! I don't think "grouping them together" would mislead anyone, either. As you say, it reinforces the point. The cool thing about "on the same page," though, is its specificity.

Reply:
Perfect! Added both edits into a batch so I can commit together once we resolve.